### PR TITLE
web: add support of internatialization (#11)

### PR DESCRIPTION
**Pull request by nikolaas**
# Proposed Changes

The PR adds support for the internationalization of the login page.

### CODE REVIEW

File: internal/web/web.go [hunk 1]
```diff
@@ -111,16 +150,31 @@ func (r *HTMLRenderer) RenderTemplate(w http.ResponseWriter, name string, data i
 	w.Header().Set("Content-Type", "text/html; charset=utf-8")
 	_, err = buf.WriteTo(w)
 	return err
+}
 
+func isOldStyleUserTemplate(root *template.Template) bool {
```
Comment (konstlepa) [inline internal/web/web.go:155]: Mention the issue that causes the creation of this function.

File: internal/web/web.go [hunk 2]
```diff
@@ -89,20 +96,52 @@ func (r *HTMLRenderer) RenderTemplate(w http.ResponseWriter, name string, data i
...
-	if err != nil {
-		return errors.Wrapf(err, "failed to clone the main template for template %q: %s", name, err)
-	}
-	t, err = t.Parse(string(b))
+	root, err := template.New("main").Parse(string(b))
 	if err != nil {
 		return errors.Wrapf(err, "failed to parse template %q: %s", name, err)
 	}
 
+	if isOldStyleUserTemplate(root) {
```
Comment (konstlepa) [inline internal/web/web.go:104]: Mention that this code will be deprecated in the future major release.

File: internal/web/web_test.go [hunk 3]
```diff
@@ -21,6 +21,97 @@ import (
 )
 
 func TestHTMLRenderer(t *testing.T) {
```
Comment (konstlepa) [inline internal/web/web_test.go:23]: Merge functions TestHTMLRender and TestHTMLRenderer_old_template.

**Review by konstlepa — CHANGES_REQUESTED** [review]

File: internal/web/web.go (hunk 1 above)
Comment (nikolaas) [reply internal/web/web.go:166]: Done

File: internal/web/web.go (hunk 2 above)
Comment (nikolaas) [reply internal/web/web.go:108]: Done

File: internal/web/web_test.go (hunk 3 above)
Comment (nikolaas) [reply internal/web/web_test.go:83]: Done

Comment (konstlepa) [reply internal/web/web_test.go:83]: Remove the dead code.

**Review by konstlepa — CHANGES_REQUESTED** [review]

**Pull request merged**